src/core/Tabs/style.tsx

```diff
@@ -75,16 +75,11 @@ export const StyledTab = styled(RawTab, {
     const spaces = getSpaces(props);
     const fontWeights = getFontWeights(props);
 
-    const { sdsSize } = props;
-
-    const isLarge = sdsSize === "large";
-
     return `
       font-weight: ${fontWeights?.semibold};
       margin: 0 ${spaces?.xl}px ${spaces?.xxs}px 0;
 
       // (thuang): Large Tab height is 30px, the offset is 4px
-      height: ${isLarge ? 26 : 22}px;
```

Comment:
why was this deleted? Also, if it should be deleted the comment should be as well

Reply:
Same rational as the `InputDropdown`. The height should be implicitly calculated from the line height and padding rather than hard-coded. 

Thanks for pointing out the comment. I should have deleted that.